authors/apps/authentication/serializers.py

```diff
@@ -1,10 +1,22 @@
 from django.contrib.auth import authenticate
-
+from django.core.validators import RegexValidator
 from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
 
 from .models import User
 
 
+def validate_password(password):
+    """
+    Validates if a string is alphanumeric
+    :return: given string or raises a serializers.ValidationError
+    """
+
+    return RegexValidator(
```

Comment:
We would need to change the Regex validator for the password, as it currently is it only allows alphanumeric characters. The password should have **at least** both alphabetic and numeric characters to be a valid password but should also allow for special characters which make a stronger password